### ISSUE TITLE
QA

### ISSUE THREAD

**Issue by Keyrxng** [ubq-testing/ask-plugin#1]
Relevant PR: https://www.github.com/ubiquity-os-marketplace/command-ask/pull/1

Demoing the effectiveness

**Comment by Keyrxng** [comment]
@ubqbot create a table of the context you are aware of, ensure you include "www." in any url references to any issue or PR, prefer using plaintext titles instead

**Comment by ghost** [comment]
Here is a table summarizing the context I am aware of:

| Type        | ID  | Repository                            | Title/Description                                                                                    |
|-------------|-----|---------------------------------------|------------------------------------------------------------------------------------------------------|
| Pull Request| 1   | ubiquity-os-marketplace/command-ask    | Implements `ask-gpt` command, adds OpenAI support, removes Supabase integration.                       |
| Issue       | 29  | ubiquibot/plugins-wishlist            | Context-aware question support for GPT-4o integration, with linked conversation context.              |
| Issue       | 73  | ubiquity-os/ubiquity-os-kernel        | Discusses `runsOn` for defining plugin event subscriptions in configuration manifests.                |
| Issue       | 75  | ubiquity-os/ubiquity-os-kernel        | Modification for the kernel to support multiple commands in the configuration like `start stop`.      |
| Issue       | 25  | ubq-testing/bot-ai                    | Resolves issue #16, furthering discussion around improvements to the chatbot.                         |
| Issue       | 29  | ubq-testing/ask-plugin                | No specification available.                                                                          |
| Issue       | 16  | ubq-testing/bot-ai                    | Initial setup for database integration with Supabase for chatbot's memory system.                     |
| Issue       | 15  | ubq-testing/ask-plugin                | No specification available.                                                                          |

Each entry includes the repository name and type (pull request or issue) along with a brief description.

<!--
{
  "output": 320,
  "input": 54413,
  "total": 54733
}
--!>

**Comment by 0x4007** [comment]
Did it crawl all the links in the other pulls? If so, thats really cool. 

>  
> Issue | 15 | ubq-testing/ask-plugin | No specification available.
> -- | -- | -- | --
> 

Is this a hallucination or did you delete this?

**Comment by Keyrxng** [comment]
Yeah that issue was deleted. I wrapped the fetch for issue and comments and return null as opposed to extracting the error and feeding that to the LLM but I guess it could be done?

> Did it crawl all the links in the other pulls? If so, thats really cool.

Yeah every referenced issue/PR should be pulled into context. As seen here one URL to a single PR was 54k tokens in context and referenced 8 issues
